notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>The assumption of using the first instance from the list seems problematic. Should we just use <code>tensorboard.resource_name</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Yeah let's just use the name derived from the previous step at </p><p>TENSORBOARD_RESOURCE_NAME = tensorboard.gca_resource.name</p>